constraints.txt

```diff
@@ -37,8 +41,10 @@ pbr==1.10.0
 positional==1.1.1
 prettytable==0.7.2
 pyasn1==0.2.3
+pycodestyle==2.3.1
 pycparser==2.17
 pycrypto==2.6.1
+pyflakes==1.5.0
```

Comment:
Those are the constraints from test-requirements, as we don't have a test-constraints file.